capstone/src/components/User/Adventurer/Preference/FilterArea/FilterArea.jsx

```diff
@@ -18,6 +18,7 @@ export default function FilterArea({
         <HStack spacing="40px" mb="20px">
           <Checkbox ml="45px" onChange={(e) => setShowMinimumValues(e.target.checked)} />
           <Badge> {priority} {displayText} </Badge>
+          <Button rightIcon={<DeleteIcon />} colorScheme="red" variant="outline" alignSelf="right">Delete</Button>
```

Comment:
I've mentioned this before, but we should try to reduce the amount of repeated markup for several reasons:

* If we decide to make a change here it can be easy to accidentally forget to update both places. Notice that by adding the Delete button, you needed to add it in two places.
* It's hard to tell what the difference is just by scanning it. I guess there is something with the badge?

Try something like this instead:

```
const badge = showMinimumValues ? <Badge colorScheme="yellow"> {sliderValue} <DefineIcon units={units} />  or Higher </Badge> : <Badge> {priority} {displayText} </Badge>;
return (
  <Box>
     <HStack ..>
        <Checkbox onChange={} />
        {badge}
        <Button ...>Delete</Button>
    </HStack>
    ....
```

Reply:
Thanks a lot for that comment! I still need to work on my skills to refactor my code so that I don't repeat my code, I appreciate a lot you bring it up again